src/polar/Pie.tsx

```diff
@@ -402,7 +402,8 @@ export class Pie extends PureComponent<Props, State> {
       return option(props);
     }
 
-    return <Curve {...props} type="linear" className="recharts-pie-label-line" />;
+    const className = clsx('recharts-pie-label-line', typeof option !== 'boolean' ? option.className : '');
```

Comment:
nit: we kind of already have a typeguard for this right? 
https://github.com/recharts/recharts/commit/76bbdcb6d4c8e13f73f8ca7834bcf2a28143ac8d#diff-0030e00d80866b12434b3def516dcf03f12a37360fcaaa5fc6a7c32e0d04d17cR187

We can move the typeguard out of polar utils to somewhere that is more generic, and then make the function more generic so it works for every case.

And actually to make it a true "type guard" we could instead do

```tsx
export const hasCustomClassName = <T extends SVGElement>(
  option?: SVGProps<T> | ReactElement<SVGElement> | ((props: any) => ReactElement<SVGElement>) | boolean,
): option is { className: string } =>
  !isValidElement(option) && !isFunction(option) && typeof option !== 'boolean' && 'className' in option;
```

Then `const className = clsx('recharts-pie-label-line', hasCustomClassName(option) ? option.className : '');